update_check.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Please delete this file.

Reply:
![image](https://user-images.githubusercontent.com/42309414/78334172-e0dbee80-75bd-11ea-92b4-8c30d3af874e.png)

Reply:
Done